<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogmanage</class>
 <widget class="QDialog" name="Dialogmanage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>644</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>644</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>房间管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hotel.qrc">
    <normaloff>:/new/prefix1/Images/Common/LOGO.png</normaloff>:/new/prefix1/Images/Common/LOGO.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>621</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/Images/Common/manage.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>601</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>房价管理</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>单人间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>标准间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>商务间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>豪华套房</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>总统套房</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_singleoriginal">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string> 原 价/元</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>修改后房价/元</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_singlechange">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_presidentoriginal">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_deluxeoriginal">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_bussinessoriginal">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_standardoriginal">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_bussinesschange">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_standardchange">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_deluxechange">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>160</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_presidentchange">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>200</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_yesforprice">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_noforprice">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>591</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>*注：只能修改空闲房间的房价，对于已有人入住的房间，当退房后再自动进行房价修改</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>空闲-&gt;维修</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>需要维修的房间</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_serviceroomnumber">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> * 注：只有空闲房才能转成维修状态</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_yesforservice">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_noforservice">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>维修-&gt;空闲</string>
    </attribute>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>完成维修的房间转为空闲</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_changefreenumber">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> * 注：状态为维修的房间才能转为空闲</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_yesforfree">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_noforfree">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="hotel.qrc"/>
 </resources>
 <connections/>
</ui>
